docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -6,6 +6,16 @@ The MySQL connector allows querying and creating tables in an external
 `MySQL <https://www.mysql.com/>`_ instance. This can be used to join data between different
 systems like MySQL and Hive, or between two different MySQL instances.
 
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a MySQL data
```

Comment:
GDDSD suggests the use of [active voice](https://developers.google.com/style/voice?hl=en):

To connect to a MySQL data source, the following requirements must be met: